src/ucp/wireup/wireup_cm.c

```diff
@@ -1157,7 +1187,8 @@ ucp_ep_cm_server_create_connected(ucp_worker_h worker, unsigned ep_init_flags,
         goto err_destroy_ep;
     }
 
-    ucp_ep_update_remote_id(ep, conn_request->sa_data.ep_id);
+    sa_data = (ucp_wireup_sockaddr_data_base_t*)(conn_request + 1);
```

Comment:
`sa_data[0]` field could simplify the code a bit

Reply:
maybe no need to cast